src/atoms/Stepper/Stepper.story.tsx

```diff
@@ -0,0 +1,63 @@
+import { storiesOf } from '@storybook/react';
+import React, { Component } from 'react';
+
+import Button from '../Button';
+import Stepper from './Stepper';
+
+const beforeStep = <h2>Start Stepping!</h2>;
+const stepOne = <h2>Step One</h2>;
+const stepTwo = <h2>Step Two</h2>;
+const stepThree = <h2>Step Three</h2>;
+const afterStep = <h2>All steps are done!</h2>;
+const steps = [beforeStep, stepOne, stepTwo, stepThree, afterStep];
+
+// tslint:disable-next-line: no-empty-interface
+interface StepActionProps {}
+
+interface StepActionState {
+  step: number;
+  total: number;
+}
+
+class Step extends Component<StepActionProps, StepActionState> {
+  public state: StepActionState = { step: 0, total: steps.length };
+
+  constructor(props: StepActionProps) {
```

Comment:
The `handleClick` functions can be changed to use the arrow syntax which will allow you to remove the constructor since you won't have to bind them.